static/app/views/alerts/issueRuleEditor/ruleNode.tsx

```diff
@@ -81,7 +81,7 @@ class RuleNode extends React.Component<Props> {
     //
     // However there are integrations that give the form field choices with the value as number, but
     // when the integration configuration gets saved, it gets saved and returned as a string
-    const choices = fieldConfig.choices.map(([key, value]) => [`${key}`, value]);
+    const options = fieldConfig.choices.map(([value, label]) => ({value, label}));
```

Comment:
changed the names here cuz it was a little misleading, value is the first index of the array and the label is the second